appveyor.yml

```diff
@@ -1,20 +1,33 @@
 environment:
   SAXON_CP: '%TEMP%\xspec\saxon\saxon9he.jar'
   matrix:
-    # latest Saxon 9.7 version
+    # latest Saxon 9.7 version and XML Calabash
     - SAXON_VERSION: 9.7.0-15
+      XMLCALABASH_VERSION: 1.1.15-97

```

Comment:
Thanks for the clarification, it makes sense to skip this test for other versions of Saxon. Shall I implement something similar in [xspec.bats](https://github.com/xspec/xspec/blob/master/test/xspec.bats)? i.e skip the test if the version of Saxon is not  9.7.0.15 or XMLCalabash is not installed.